copyparty/web/tl/vie.js

```diff
@@ -0,0 +1,723 @@
+	"wt_pst": "dán một lựa chọn đã cắt / sao chép trước đó$NPhím tắt: ctrl-V",
+	"wt_selall": "chọn tất cả các tệp$NPhím tắt: ctrl-A (khi tệp được chọn)",
+	"wt_selinv": "đảo ngược lựa chọn",
+	"wt_zip1": "tải thư mục này dưới định dạng nén",
+	"wt_selzip": "tải lựa chọn dưới định dạng nén",
+	"wt_seldl": "tải lựa chọn dưới dạng các tệp riêng biệt$NPhím tắt: Y",
+	"wt_npirc": "sao chép thông tin bản nhạc theo định dạng irc",
+	"wt_nptxt": "sao chép thông tin bản nhạc dưới dạng văn bản thuần túy",
+	"wt_m3ua": "thêm vào danh sách phát m3u (bấm <code>📻copy</code> sau)",
+	"wt_m3uc": "sao chép danh sách phát m3u vào bảng nhớ tạm",
+	"wt_grid": "chuyển đổi chế độ xem danh sách / lưới $NPhím tắt: G",
+	"wt_prev": "bài trước$NPhím tắt: J",
+	"wt_play": "phát / tạm dừng$NPhím tắt: P",
+	"wt_next": "bài sau$NPhím tắt: L",
+
+	"ul_par": "tải lên song song:",
+	"ut_rand": "ngẫu nhiên hoá tên tệp",
+	"ut_u2ts": "sao chép dấu thời gian chỉnh sửa cuối$Ntừ hệ thống tệp của bạn lên máy chủ\">📅",
+	"ut_ow": "ghi đè các tệp đã có trên máy chủ?$N🛡️: không bao giờ (sẽ tạo tên tệp mới)$N🕒: ghi đè nếu tệp trên máy chủ cũ hơn$N♻️: luôn ghi đè nếu hai tệp khác nhau",
+	"ut_mt": "tiếp tục hash các tệp khác trong khi tải lên$N$NCó thể tắt nếu CPU hoặc HDD của bạn bị nghẽn",
+	"ut_ask": 'yêu cầu xác nhận trước khi bắt đầu tải lên">💭',
+	"ut_pot": "cải thiện tốc độ tải lên trên các thiết bị chậm$Nbằng cách đơn giản hoá giao diện người dùng",
+	"ut_srch": "không tải lên, chỉ kiểm tra xem tệp$Nđã tồn tại trên máy chủ hay chưa (sẽ quét toàn bộ thư mục bạn có quyền đọc)",
+	"ut_par": "tạm dừng tải lên bằng cách đặt thành 0$N$NTăng lên nếu kết nối chậm hoặc độ trễ cao$N$NGiữ ở mức 1 khi dùng LAN hoặc nếu ổ cứng máy chủ bị nghẽn",
+	"ul_btn": "thả tệp / thư mục<br>ở đây (hoặc nhấn vào tôi)",
+	"ul_btnu": "T Ả I  L Ê N",
+	"ul_btns": "T Ì M  K I Ế M",
+
+	"ul_hash": "hash",
+	"ul_send": "gửi",
+	"ul_done": "hoàn tất",
+	"ul_idle1": "chưa có mục nào trong hàng chờ tải lên",
+	"ut_etah": "tốc độ &lt;em&gt;hash&lt;/em&gt; trung bình và thời gian dự kiến để hoàn tất",
+	"ut_etau": "tốc độ &lt;em&gt;tải lên&lt;/em&gt; trung bình và thời gian dự kiến để hoàn tất",
+	"ut_etat": "tốc độ &lt;em&gt;tổng&lt;/em&gt; trung bình và thời gian dự kiến để hoàn tất",
+
+	"uct_ok": "hoàn tất thành công",
+	"uct_ng": "không hợp lệ: lỗi / bị từ chối / không tìm thấy",
+	"uct_done": "đã xử lý: gồm cả thành công và không hợp lệ",
+	"uct_bz": "đang hash hoặc tải lên",
+	"uct_q": "nhàn rỗi, đang chờ",
+
+	"utl_name": "tên tệp",
+	"utl_ulist": "danh sách",
+	"utl_ucopy": "sao chép",
+	"utl_links": "đường dẫn",
+	"utl_stat": "trạng thái",
+	"utl_prog": "tiến trình",
+
+	// keep short:
+
+	// phần up2k
+	"utl_404": "404",
+	"utl_err": "LỖI",
+	"utl_oserr": "Lỗi hệ thống",
+	"utl_found": "tìm thấy",
+	"utl_defer": "hoãn",
+	"utl_yolo": "YOLO",
+	"utl_done": "hoàn tất",
+
+	"udt_up": "Tải lên",
+	"udt_srch": "Tìm kiếm",
+	"udt_drop": "thả vào đây",
```

Comment:
the strings `ul_flagblk` and `ul_btnlk` are missing, and they were replaced with another copy of these three strings